test/integration/virtual-rules/area-alt.js

```diff
@@ -1,4 +1,4 @@
-describe('area-alt', function() {
+describe('area-alt virtual-rule', function() {
```

Comment:
Without the additional `virtual-rule` this describe name would collide with describes from the `checks` directory and not output any describe blocks, making this a lot harder to know what tests this was from.